engine/servermaster/server.go

```diff
@@ -524,6 +525,19 @@ func (s *Server) registerMetaStore() error {
 	if err != nil {
 		return err
 	}
+	if cfg.BusinessMetaConf.StoreType == metaModel.StoreTypeSQL {
+		// Normally, a schema will be created in advance and we may have no privilege
+		// to create schema for business meta.
+		// Just for easy test here. Ignore any error.
+		ctx, cancel := context.WithTimeout(ctx, 1*time.Second)
+		defer cancel()
+		if err := meta.CreateSchemaIfNotExists(ctx, *(s.cfg.BusinessMetaConf)); err != nil {
```

Comment:
If create schema fails, when will it be created?

Reply:
It need to be created in advance.
For easy test, may be it's better to and some retry here.

Reply:
All meta operations have no retry now.
I will add retry logic in other pr.